### ISSUE TITLE
easy way to upvote crossposts 

### ISSUE THREAD

**Issue by Die4Ever** [LemmyNet/lemmy-ui#2825]
### Requirements

- [X] This is a feature request and not a bug report. Otherwise, please create a new [bug report](https://github.com/LemmyNet/lemmy-ui/issues/new?assignees=&labels=bug%2Ctriage&projects=&template=BUG_REPORT.yml) instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this request (or a similar one) already exists.
- [X] It's a single feature. Please don't request multiple features in one issue.

### Describe the feature you'd like

Would be nice to have an easy way to upvote all the (recent?) crossposts without having to visit each one. I saw a complaint about post de-duplication causing the upvotes to be cannibalized.

https://sopuli.xyz/comment/13225378

**Comment by dessalines** [comment]
I don't think this is desirable. It'd either require a special *mass-upvote* button just for this purpose, or expanding the cross-post listings for each post to show a vote button. I agree with the person in that thread that posts should stand on their own, and go to them specifically to upvote.

Also we do have scheduled posts merged, it'll be in the next major release.
